<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Кофий</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>851</width>
      <height>591</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>6</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>142</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>70</number>
    </attribute>
    <column>
     <property name="text">
      <string>Название сорта</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Степень обжарки</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Молотый/в зернах</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Описание вкуса</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Цена, руб</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Объём упаковки, мл</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="button_Edit">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>20</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_Add">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>90</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1095</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
